src/main/java/com/sphy/hotelmanagementapplication/bootstrap/BootStrapData.java

```diff
@@ -55,42 +62,56 @@ public void run(String... args) throws Exception {
         admin.getHotels().add(ksenia);
         adminRepository.save(admin);
 
+        Set<HotelAmenity> amenitySet = new HashSet<>();
+
+        HotelAmenity amenity1 = new HotelAmenity();
+        //amenityHotelRepository.save(amenity1);
+        amenity1.setAmenitiesH(HotelAmenity.AmenitiesHotel.AIRPORTTRANSPORT);
+
+        HotelAmenity amenity2 = new HotelAmenity();
+       // amenityHotelRepository.save(amenity2);
+        amenity2.setAmenitiesH(HotelAmenity.AmenitiesHotel.GYM);
+
+        HotelAmenity amenity3 = new HotelAmenity();
+        //amenityHotelRepository.save(amenity3);
+        amenity3.setAmenitiesH(HotelAmenity.AmenitiesHotel.PETSALLOWED);
+
+       // ksenia.getHotelAmenity().add(amenity1);
+       // ksenia.getHotelAmenity().add(amenity2);
+       // ksenia.getHotelAmenity().add(amenity3);
+        amenityHotelRepository.saveAll(amenitySet);
```

Comment:
Before save amenitySet you mast add into it amenity1, amenity2, amenity3 because otherwise is empty.